libraries/AP_Notify/Display_SSD1306_I2C.cpp

```diff
@@ -22,33 +22,27 @@ static const AP_HAL::HAL& hal = AP_HAL::get_HAL();
 
 bool Display_SSD1306_I2C::hw_init()
 {
-    // display init sequence
-    uint8_t init_seq[] = {0xAE, 0xD5, 0x80, 0xA8, 0x3F, 0xD3,
+    _dev = hal.i2c_mgr->get_device(2, SSD1306_I2C_ADDRESS);
```

Comment:
@lgywata In long term we should move that into a constructor to make it more generic.